CourseApp/Table.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TableProj
+{
+    public class Table
+    {
+        public int width { get; set; }
+        public int height { get; set; }
+        public string wood { get; set; }
+
+        public Table()
+        {
+            this.width = 0;
+            this.height = 0;
+            Console.WriteLine("Now you have abstract table");
+        }
+        public Table(int width, int height)
+        {
+            this.width = width;
+            this.height = height;
+        }
+
+        public int areaOfTable()

```

Comment:
If it's returns int - than it's not necessary to do any Console/Writlines  (Single Responcibility principle)